components/tools/OmeroPy/test/integration/test_iquery.py

```diff
@@ -150,3 +151,16 @@ def testQueryTaggedUnique(self):
         for idx in range(len(result1)-1):
             # Omit final since == isn't defined for Ice objects.
             assert result1[idx] == result2[idx]
+
+    def testClassType(self):
+        created = []
+        for Ann in (CommentAnnotationI, TagAnnotationI):
```

Comment:
This was mostly to show that nothing weird was happening in terms of mapping (statics in `IceMapper` etc.). I don't expect there to be any value in adding more.